aphrodite/engine/ray_tools.py

```diff
@@ -5,6 +5,7 @@
 from aphrodite.common.config import ParallelConfig
 from aphrodite.common.logger import init_logger
 from aphrodite.common.utils import get_open_port, is_hip
+import os, sys
```

Comment:
Can you split these into two separate imports, and place them at the top of the file (since they're stdlib)?